src/main/java/kostka/moviecatalog/entity/Rating.java

```diff
@@ -1,12 +1,23 @@
 package kostka.moviecatalog.entity;
 
 public class Rating {
+    private Long ratingId;
     private Long movieId;
-    private int value;
+    private Long userId;
+    private int ratingValue;
 
-    public Rating(final Long movieId, final int value) {
-        this.movieId = movieId;
-        this.value = value;
+    public Rating(final Long ratingId, final Long movieId, final Long userId, final int ratingValue) {
```

Comment:
proc ma ten konstruktor parametry, kdyz se nikam nekladaji?

Reply:
a jo, vygenerovany pres intelij a pak jsem to nejak odmazal, dobry postreh